src/js/containers/recipient/topFive/TopFiveContainer.jsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * TopFiveContainer.jsx
+ * Created by Kevin Li 5/15/18
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { isCancel } from 'axios';
+
+import {
+    getTrailingTwelveMonths,
+    convertFYToDateRange
+} from 'helpers/fiscalYearHelper';
+import * as SearchHelper from 'helpers/searchHelper';
+import BaseStateCategoryResult from 'models/v2/state/BaseStateCategoryResult';
+
+import TopFive from 'components/recipient/topFive/TopFive';
+
+const propTypes = {
+    total: PropTypes.number,
+    category: PropTypes.string,
+    fy: PropTypes.string
+};
+
+export class TopFiveContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            loading: true,
+            error: false,
+            results: []
+        };
+
+        this.request = null;
+    }
+
+    componentDidMount() {
+        this.loadCategory();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.fy !== this.props.fy) {
+            this.loadCategory();
+        }
+    }
+
+    dataParams() {
+        let timePeriod = null;
+        if (this.props.fy === 'latest') {
+            const trailing = getTrailingTwelveMonths();
+            timePeriod = {
+                start_date: trailing[0],
+                end_date: trailing[1]
+            };
+        }
+        else if (this.props.fy !== 'all' && this.props.fy) {
+            const range = convertFYToDateRange(parseInt(this.props.fy, 10));
+            timePeriod = {
+                start_date: range[0],
+                end_date: range[1]
+            };
+        }
+
+        const filters = {
+            place_of_performance_scope: 'domestic',
```

Comment:
We should be filtering on recipient instead of location here using `recipient_search_text` (which will search DUNS or recipient name)
See documentation for the standard filter object: https://github.com/fedspendingtransparency/usaspending-website/blob/dev/api_contracts/contracts/search/AdvancedSearch.md#filterobject-object